js_modules/dagster-ui/packages/ui-core/src/ticks/SensorDryRunDialog.tsx

```diff
@@ -155,7 +189,9 @@ const SensorDryRun = ({repoAddress, name, currentCursor, onClose, jobName}: Prop
         </Box>
       );
     }
-  }, [sensorExecutionData, error, submitting, onClose, submitTest]);
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
`onLaunchAll` (https://github.com/dagster-io/dagster/pull/25633/files#diff-515db7eaf248b881449756acdc385f71173633288183df1f034c7c968657fc52R141) was a required dep because it was getting called in the onClick. Didn't want that to cause rerendering so added that rule

Reply:
make `onLaunchAll` use `useCallback`

Reply:
https://github.com/dagster-io/dagster/pull/25633/files#diff-515db7eaf248b881449756acdc385f71173633288183df1f034c7c968657fc52R141